doc/adr/0001-mock-api-endpoints-in-dev.md

```diff
@@ -0,0 +1,29 @@
+# 1. Provide optional mocked API endpoints in development
+
+Date: 2023-05-16
+
+## Status
+
+Accepted
+
+## Context
+
+We've been blocked by setbacks in API development for this project, and we want
+UI development to be able to continue (both now and in any similar situations in
+the future) regardless of the status of API development.
+
+## Decision
+
+We already use Wiremock for stubbing tests in the test environment. We will
+extend this to make Wiremock available for use in development, as well as adding
+some scripts to stub the API endpoints we want to use. These will run when we
+start the app with a given flag, but will also be able to be run at will.
+
+The scripts to start the server or just the backing services will accept a flag
+to mock the API, which will allow us to toggle between using using canned
+response fixtures in the UI and connecting to a running API backend application.
+
+## Consequences
+
+This will unblock frontend work and prevent future blocks should there be any
```

Comment:
Might be worth adding a consequence being "We'll need to make sure we keep the data structures of the API and UI in sync when relying on a mock server, but we believe we can be alerted of any changes through contract testing with Pact and, where they exist, generating types from the API."